protocols/v2/channels-sv2/src/share_accounting.rs

```diff
@@ -0,0 +1,47 @@
+use bitcoin::hashes::sha256d::Hash;
+
+pub trait ShareAccountingTrait {
```

Comment:
```suggestion
pub trait ShareAccounting {
```

no need to have `Trait` in the name